about.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">                  
+    <link rel="stylesheet" href="mobile-about.css" />
+    <link rel="stylesheet" href="desktop-about.css" />
+    <title>About</title>
+</head>
+<body>
+    <header class="black-menu hide">
+        <ul class="black-menu hide">
+          <li><a href="#"><img class="hide link" src="Images/facebook.png" alt="facebook-logo" height="20"/></a></li>
+          <li><a href="#"><img class="hide link" src="Images/twitter.png" alt="twitter-logo" height="20"/></a></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </header>
+    
+
+      <div class="menu">
+          <a href="#" class="disapeer" id="hamburger-menu" ><img class="disapeer" src="Images/menu-mobile.png" alt="hamburger-menu" width="60" /></a>
+  
+          <ul class="menu-section">
+              <li>
+                <span class="material-icons close disapeer">close</span>
+              </li>
+              <li><a href="index.html" class="link hide"><img class= "hide right" src="Images/logo-villa.png" alt="logo" width="135" /></a></li>
+              <li><a href="about.html">About</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Join</a></li>
+              <li><a href="#">Sponsor</a></li>
+              <li><a href="#">News</a></li>
+              <li class="cc"><a href="#">CC Campaign</a></li>
+            </ul>
```

Comment:
- [ ] OPTIONAL
Your code is not properly indented. Kindly ensure that your code has consistent **spacing** for indentation. The recommended spacing is **2 spaces**. You can install and configure a code formatter plugins such as [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) or [Beautify](https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify) in your code editor for formatting your code to ensure consistent and well-indented code.